src/hypothesis/strategies.py

```diff
@@ -804,9 +804,8 @@ def dm_func(denom):
 
 @cacheable
 @defines_strategy
-def decimals(
-    min_value=None, max_value=None, allow_nan=None, allow_infinity=None
-):
+def decimals(min_value=None, max_value=None, allow_nan=None,
+             allow_infinity=None, places=None, precision=None):
```

Comment:
It's possible to construct an equivalent strategy either way (as I've written you'd use flatmap, otherwise set bounds to the same value), and I did consider it.  The deciding factor for me was that I think users will want a specific level of precision more often than a bounded range, and three interacting boundary conditions felt like a really clumsy interface.